src/lib/components/ButtonGroup/README.mdx

```diff
@@ -43,6 +43,12 @@ See [API](#api) for all available options.
 - Use button group to group **related actions** that a user can take. Buttons
   should not be grouped just to save space on the screen.
 
+- **Mixing multiple button priorities** within a button group is **not
```

Comment:
Priorities can not be mixed, they come from context:

[react-ui/src/lib/components/Button/Button.jsx](https://github.com/react-ui-org/react-ui/blob/296e73fd8d8525eaffeac9bc245a3125986625e2/src/lib/components/Button/Button.jsx#L42)

There is no need to warn users about it.

Reply:
I'm pointing it out because I tried to do that, it didn't work and I wasn't sure if it was a bug or feature.

Reply:
OK, but then we should also mention the other enforced props like `size` and `block`.